.gitignore

```diff
@@ -1,10 +1,6 @@
-# This gitignore file is for this projects concerns.
-# Please set swap files and system folder files for your operating system and
-# editors to be ignored in
-# the 'global ignore file'.
-# See: http://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration
-#    : search for "global .gitignore"
-
 # bundler specific
 .gem
 .bundle
+
+# Pages index is not staged
+pages/index.json
```

Comment:
I did not remove it from `.gitignore`. I added it to `.gitignore`.

So you will never see it in `git status`.